nixos/modules/services/security/keylime-agent.nix

```diff
@@ -0,0 +1,83 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.keylime-agent;
+in
+{
+  options = {
+    services.keylime-agent = {
+      enable = mkOption {
+        default = false;
+        type = types.bool;
+        description = lib.mdDoc "Whether to enable keylime-agent service.";
+      };
+
+      stateDir = mkOption {
+        type = types.path;
+        default = "/var/lib/keylime/";
+        defaultText = literalExpression "/var/lib/keylime/";
+        description = lib.mdDoc "State directory for keylime";
+      };
+
+      user = mkOption {
+        type = types.str;
+        default = "keylime";
+        description = lib.mdDoc "User which runs keylime.";
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "tss";
+        description = lib.mdDoc "Group which runs keylime.";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.etc."keylime/agent.conf".source = "${pkgs.rust-keylime}/etc/keylime/agent.conf";
+
+    systemd.tmpfiles.rules = [
+      "d ${cfg.stateDir} 0770 ${cfg.user} ${cfg.group} - -"
+      "d ${cfg.stateDir}/cv_ca 0770 ${cfg.user} ${cfg.group} - -"
+    ];
+
+    systemd.services.keylime-agent = {
```

Comment:
The package seems to vendor these files. Maybe we should use the upstream files, instead of maintaining our own copy?